### ISSUE TITLE
Open Grant Proposal: NFTsForPolitics

### ISSUE THREAD

**Issue by Ciudadana** [filecoin-project/devgrants#758]
# Open Grant Proposal: **NFTsForPolitics**

**Proposal Category:**  `integration-adoption`

**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT, APACHE2, or GPL licenses?:**

Yes. 

## Project Description

Politicians and political campaigns worldwide have two specific problems: lack of connection with the youth, who feel alienated from politics, and the bureaucracies (and obscurity) of the current political campaign fundraising processes, which not only is bad in itself but also delegitimizes politics in the public view. 

With NFTForPolitics we aim to solve both of those issues, providing our counterparts the transparency of the blockchain to do fundraising but also the ability to connect with that younger audience through digital art. 

We already have significant interest from clients in Uruguay and Panamá, and aim to continue developing our footprint to make this a global project and educate the world of politics on the benefits of decentralized systems such as IPFS at providing transparency to the public. 

## Value

**Benefits**

We believe this proposal is valuable for the Filecoin ecosystem because it uses Filecoin technology (mostly IPFS) to onboard tens of thousands of users to crypto, and legitimize crypto in the world of politics, generating the first platform to power crypto campaigns worldwide in an approach that isn’t solely financially related. 

We think that the Protocol Labs ecosystem presents a lot of opportunities to the world of politics, creating the possibility of combating fake news against politicians by backing their official statements in IPFS or creating decentralized campaign finance records so that everyone can know where their campaign contributions are flowing. 

**Risks**

There definitely are risks if we don’t get it right, mostly communicational risk both for NFTForPolitics and our counterparts, who, if things go wrong, could be accused of utilizing crypto to speculate or to help launder money to campaign financing. We believe that with our tech team and more than 10 years of experience in political communication, these risks can be minimized enough to make the overall opportunity attractive, but they do exist. 

We identify two main risks on this project: to begin, economic context. We are in the midst of a financial turmoil that has been particularly cruel with the crypto market, and even though that can change in a couple of months, we cannot be sure we won't be on a bear market for 18-24 months, which poses a challenge in implementing the campaigns and actually generating demand for the NFTs, as we have seen general NFT volume go down significantly. The other risk is politicians being skeptic of the platform or the general ecosystem (this risk is maximized in this state of the market) or not understanding the benefits of utilizing it and though we have had good reception so far with the politicians themselves (in the cases of Panama and Uruguay, for example), it will be hard to transform the traditional fundraising mechanisms that have their own intrinsic incentives and power dynamics. 

We have thought extensively about the risks (since our company's reputation is our biggest asset and it is on the line with this project) and we believe that the risk-reward is really attractive, so we decided to move forward. 

## Deliverables

When the project is finished, we will have a functional site that has different features for politicians and advisors to: a) understand decentralized technologies and how they function, specifically IPFS as a way to create a unique source of truth for communication and fundraising records, and NFTs (firstly in Solana but open tu being multichain in the future) as a way to create digital collectibles, connect with their communities and receive micro-funding from their supporters. 

## Development Roadmap

1. Short online course with tutorials in English and Spanish to explain crypto in layman terms from a political perspective, aimed at politicians and campaign teams. (5k -- 3 weeks). 

2. Ability to create a test NFT and see a preview easily in a drag and drop fashion, to have campaign teams or candidates easily create test NFTs and then have a  contact button that onboards them to publish their NFT and get in touch with our team to develop the communication strategy. (7k -- 2 weeks).

<img width="596" alt="Captura de Pantalla 2022-07-12 a la(s) 12 26 56" src="https://user-images.githubusercontent.com/107049570/178539839-9bab3123-7e5e-4350-abdd-a402b3b90fb1.png">

<img width="582" alt="Captura de Pantalla 2022-07-12 a la(s) 12 26 48" src="https://user-images.githubusercontent.com/107049570/178539924-0ba06b8d-82de-4a2f-8de8-a5065dfb9bbf.png">

<img width="572" alt="Captura de Pantalla 2022-07-12 a la(s) 12 26 37" src="https://user-images.githubusercontent.com/107049570/178539960-de703c2e-bab0-4609-8912-788e61553112.png">

This is a diagramm of how the functionality of this feature will work. 

<img width="653" alt="Captura de Pantalla 2022-07-12 a la(s) 12 26 15" src="https://user-images.githubusercontent.com/107049570/178541853-f6b8b467-7e12-447f-ab95-78409313e30c.png">

3. Build an official contributions and financial information channel in IPFS (and on a second instance adding a $FIL based incentive layer) so that all monetary inflows and outflows of the campaign are accounted for in real time and in a system that cannot be altered even by the politician or the campaign team, only achievable through IPFS and with an user friendly UI. (10k -- 2 weeks)

4. Develop an interactive map that helps navigate local electoral financial regulations regarding crypto in each country. (10k -- 2 weeks to develop but then ongoing support and information updating).

![Captura de Pantalla 2022-07-12 a la(s) 13 12 25](https://user-images.githubusercontent.com/107049570/178541779-acec45db-fe48-4cbe-8580-75e47597735e.png)

5. IPFS integration both with the NFTs and the legal identification tracker, storing all metadata from every NFT created by each of our clients in IPFS and creating an immutable change-log of the interactive map in IPFS. (5k -- 3 weeks). 

6. Create an official communications channel for each candidate/party built in IPFS to better fight against fake news and serve as a source of truth of official communication to the media and general public. The idea is for this channel to function as the official source of truth of the campaign to fight against fake news and miscommunication. (15k -- 4 weeks). 

## Total Budget Requested

**Development: 52k**

1. Short online course with tutorials in English and Spanish to explain crypto in layman terms from a political perspective, aimed at politicians and campaign teams. (5k). 

2. Ability to create a test NFT and see a preview easily in a drag and drop fashion, to have campaign teams or candidates easily create test NFTs and then have a  contact button that onboards them to publish their NFT and get in touch with our team to develop the communication strategy. (7k).

3. Build an official contributions and financial information channel in IPFS (and on a second instance adding a $FIL based incentive layer) so that all monetary inflows and outflows of the campaign are accounted for in real time and in a system that cannot be altered even by the politician or the campaign team, only achievable through IPFS. (10k)

4. Develop an interactive map that helps navigate local electoral financial regulations regarding crypto in each country. (10k)

5. IPFS integration both with the NFTs and the legal identification tracker, storing all metadata from every NFT created by each of our clients in IPFS and creating an immutable change-log of the interactive map in IPFS. (5k). 

6. Create an official communications channel for each candidate/party built in IPFS to better fight against fake news and serve as a source of truth of official communication to the media and general public. This will have all the official campaign materials such as written statements, videos, gifs, photos, etc. The idea is for this channel to function as the official source of truth of the campaign to fight against fake news and miscommunication. (15k). 

**Adoption: 20k** 

These are expenses made to drive adoption and spread of our project. The political communication and consulting world is based on conferences, which we plan to attend in order to raise awareness of the prowess of IPFS in particular and decentralized technologies for politicians and political parties. 

In August we will travel to the Dominican Republic to the [International Congress of Political Communication ](https://cumbrecp.com/congresord/) to present NFTForPolitics. 

In September there is an[ Investigative Congress of Political Communication in Malaga ](https://www.alice-comunicacionpolitica.com/congresos/ix-congreso-alice/).

In October we will travel to a [Congress of Political Communication in Buenos Aires](https://cumbrecp.com).  

We also budgeted travel to specific clients, mainly: Brazil, United States, Panamá. 

1500 USD/aero ticket on average, 9000 USD in total. 11k for accommodations, event materials, travel expenses. 

**Branding and marketing:** 35k

-Big data market research - Partnering with Idatha and using Kybalion (explained in presentation) to reach our target audience (5k). An explanation of this software is in the Ciudadana presentation in the repo links section. 

-LinkedIn Ads (5k).

-Sponsor the Buenos Aires event (10k).

-Create a sponsored talk at George Washington University about NFTs and political fundraising (https://www.gspminternational.org/). Julián Kanarek is already a collaborator and guest lecturer at GWU. (15k).

**Legal: 7k.**

-Legal fees to hire local experts and make sure we are compliant with each country's money laundering and political financing regulations to create our guide (7k).

**#Total Budget Requested: 114.000 USD.** 

## Maintenance and Upgrade Plans

We will commit our own resources + profits we plan to reinvest from our first implementations to maintain NFTForPolitics active and creating new features. All of the code will be open source with no exceptions. 

## Team

Our team has expertise in all relevant areas to make NFTForPolitics a successful project: from political communication to international relations experts, compliance lawyers and developers. 

## Team Members

These are some of our team members track record and contact info. 

-Julián Kanarek is the partner and director of Ciudadana, a company specialized in citizen, political and governmental communication.
He has Master in Communication and Culture and Bachelor in Social Communication from the Catholic University of Uruguay, he has worked in many areas of communication such as journalism, advertising, marketing, institutional communication and film.
In 2018 he was awarded at George Washington University as Political Consultant Revelation and in 2019 Digital Consultant of the Year by The Napolitan Victory Awards.
Gmail: julian@ciudadana.city
LinkedIn: https://www.linkedin.com/in/juli%C3%A1n-kanarek-garibaldi-5814821b/

-Matilde Mila holds a degree in International Relations from UDELAR. She also holds a Master's degree in International Business Administration (MBA) and a diploma in gender and public policies.
She has previously worked in private consulting services:
2018- Present Consultant at Wellness Techgroup in the technology and smart cities sector.
2016-2017- Data analyst Willis Towers Watson.
Gmail: matilde@ciudadana.city
LinkedIn: https://www.linkedin.com/in/matilde-mila-883b37a5/

-Álvaro Yañez is a graphic design technician from ORT Uruguay University. After graduating from ORT, he worked in different advertising agencies (Teorema, Notable, Escenario, Ímpetu, among others). He was the art director for clients such as Conaprole, Claro, MVOTMA, Médica Uruguaya, etc.
In the last 8 years he has directed the design department of the multimedia production company Trailer Media.
Gmail: alvaro@amen.com.uy
LinkedIn: https://www.linkedin.com/in/%C3%A1lvaro-y%C3%A1%C3%B1ez-6a11381b/

-Agustina Pérez Comenale, legal advisor, head of legal for Binance in Latam. 
Agustina is a legal expert, right now advising Binance in its LatAm expansion. She's a professor of LegalTech at the University of Montevideo, a co-organizer of the LegalTech Summit and a member of the Chamber of Blockchain and Artificial Intelligence at the Catholic University of Córdoba.
LinkedIn: https://www.linkedin.com/in/agustina-perez-comenale-452b8a4a

The development of the project will be done in collaboration with Init. Init is a software developer co-op with 30+ years of collective experience in the industry. We build digital products from scratch using many different technologies including blockchain and mobile.

The Init team es led by Federico Baña, a professional dev with over 14 years of experience that has been a key part of many successful crypto projects, like the [Space Heads NFT collection](https://spaceheads.co/), based in Solana, that sold out its initial phases, and [CryptoTracker](https://legalcryptotracker.io/), a regulatory map that has some key similarities with the map we want to build about financing campaigns in crypto. 

## Team Website

https://nftforpolitics.com/ -- https://ciudadana.city/

## Relevant Experience

This is some of our team’s highlights and relevant experience required to make this project successful. 

Julián Kanarek, partner in Ciudadana, has a proven track record as a digital political consulting, working, among others, in the campaign that ended up in the presidential victory of Carlos Alvarado in Costa Rica (who rose from 4% in the polls to winning with 61% of the vote in a span of 5 months under Julián’s watch). He also has worked in campaigns in México, Panama, Uruguay and Bolivia. All campaign strategies have been digital first, utilizing mostly social media, and NFTs and decentralized campaigns are the next natural step. 

Agustina Pérez Comenale has advised not only Binance, the biggest crypto exchange, in legal compliance in multiple countries in South America, but she also founded legalcryptotracker.io, that she allowed us to fork in order to create our interactive map of specific campaign financing regulations regarding crypto in order to properly advise our customers. 

Federico Baña is an experienced senior developer, leader of the development agency Init and lead the development of SpaceHeads (https://spaceheads.co/) a complex Solana NFT collection with features such as whitelisting, staking, a native marketplace, updating NFT metadata (reveal), etc. The collection is already in Phase 4 and has minted out. He and his team have extensive knowledge in web3 and are an integral part of the process. 

Here is a portfolio of some of Federico’s work and track record: 
[Federico Baña - Portfolio (1).pdf](https://github.com/filecoin-project/devgrants/files/9094571/Federico.Bana.-.Portfolio.1.pdf)

## Team code repositories

This is a folder with Ciudadana’s work and the rest of our team: [CIUDADANA - English Presentation.pdf](https://github.com/filecoin-project/devgrants/files/9094566/CIUDADANA.-.English.Presentation.pdf)

This is our general repository
https://github.com/Ciudadana/

This is the repository of our last project, that is also planning to use IPFS tech. 
https://github.com/Ciudadana/burstmybubble

# Additional Information

1) We learnt about the Open Grants Program through our first application to a Next Step Microgrant, which was not approved.  
2) The best email to get in touch is hola@ciudadana.city

**Comment by ErinOCon** [comment]
Hi @Ciudadana, thank you for your proposal. We will not be moving forward with this grant, but believe it could be a great fit for programs with a specific focus on funding public goods. We recommend keeping an eye out for future events at [Funding the Commons](https://fundingthecommons.io/). 

We also recommend using NFT.storage for storing and minting your NFTs!
